phoenix-pherf/cluster/pherf.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Please convert this to a python script and put it in phoenix-bin instead. All our scripts live there and are in python so they work reliably on Windows.